src/modules/footer-focus.js

```diff
@@ -0,0 +1,16 @@
+import getJQuery from '../helpers/jquery';
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'cdm:footer-focus' );
+const $ = getJQuery();
```

Comment:
Everything above here can probably be removed since it isn't being used anywhere.